html/links/manifest/mime-type-text-json-manual.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<title>Test JSON MIME Type support (text/json)</title>
+<link rel="help" href="https://html.spec.whatwg.org/#link-type-manifest" />
+<link rel="manifest" href="mime-type-text-json.webmanifest" />
+<h1>Test JSON MIME Type support for web manifest</h1>
+<p>
+  To pass, the use agent must treat the manifest valid (name is "pass"). The
+  response's Content-Type metadata is a JSON MIME type "text/json".
```

Comment:
Do we want to test an "invalid" media type?

Reply:
Good idea... will add an actual invalid type.